src/components/vm/disks/diskAdd.jsx

```diff
@@ -395,6 +395,19 @@ export const AddDiskModalBody = ({ disk, idPrefix, isMediaInsertion, vm, vms, su
     }), [defaultPool, mode, vm]);
     const storagePoolName = diskParams.storagePool?.name;
 
+    const getPoolVolumeAndDevice = (pool, volName) => {
```

Comment:
It would be more self-explanatory to name this:
getPoolFormatAndDevice

Reply:
Err yes, that's what I actually meant :see_no_evil: Updated.